From b9879b2e329a5e9e6cd6fab0a620ec7284cb6f09 Mon Sep 17 00:00:00 2001
From: Kihwal Lee <kihwal@apache.org>
Date: Wed, 13 May 2015 14:16:37 -0500
Subject: [PATCH 0653/1023] HDFS-8358. TestTraceAdmin fails. Contributed by
 Masatake Iwasaki.

(cherry picked from commit f9a46a00d2020c6d6466fbc829ada0521cb78dc0)
(cherry picked from commit 2e9392c1572e32985aaa7e073f35835bd36852bd)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tracing/TestTraceAdmin.java
---
 .../org/apache/hadoop/tracing/TestTraceAdmin.java  |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tracing/TestTraceAdmin.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tracing/TestTraceAdmin.java
index 22a4c24..acd0dbb 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tracing/TestTraceAdmin.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tracing/TestTraceAdmin.java
@@ -72,10 +72,10 @@ public void testCreateAndDestroySpanReceiver() throws Exception {
       Assert.assertEquals("ret:0, [no span receivers found]" + NEWLINE,
           runTraceCommand(trace, "-list", "-host", getHostPortForNN(cluster)));
       Assert.assertEquals("ret:0, Added trace span receiver 1 with " +
-          "configuration local-file-span-receiver.path = " + tracePath + NEWLINE,
+          "configuration dfs.htrace.local-file-span-receiver.path = " + tracePath + NEWLINE,
           runTraceCommand(trace, "-add", "-host", getHostPortForNN(cluster),
               "-class", "org.apache.htrace.impl.LocalFileSpanReceiver",
-              "-Clocal-file-span-receiver.path=" + tracePath));
+              "-Cdfs.htrace.local-file-span-receiver.path=" + tracePath));
       String list =
           runTraceCommand(trace, "-list", "-host", getHostPortForNN(cluster));
       Assert.assertTrue(list.startsWith("ret:0"));
@@ -86,11 +86,11 @@ public void testCreateAndDestroySpanReceiver() throws Exception {
       Assert.assertEquals("ret:0, [no span receivers found]" + NEWLINE,
           runTraceCommand(trace, "-list", "-host", getHostPortForNN(cluster)));
       Assert.assertEquals("ret:0, Added trace span receiver 2 with " +
-          "configuration local-file-span-receiver.path = " + tracePath + "\n",
+          "configuration dfs.htrace.local-file-span-receiver.path = " + tracePath + NEWLINE,
           runTraceCommand(trace, "-add", "-host", getHostPortForNN(cluster),
               "-class", "LocalFileSpanReceiver",
-              "-Clocal-file-span-receiver.path=" + tracePath));
-      Assert.assertEquals("ret:0, Removed trace span receiver 2\n",
+              "-Cdfs.htrace.local-file-span-receiver.path=" + tracePath));
+      Assert.assertEquals("ret:0, Removed trace span receiver 2" + NEWLINE,
           runTraceCommand(trace, "-remove", "2", "-host",
               getHostPortForNN(cluster)));
     } finally {
-- 
1.7.9.5

